.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: true
 language: python
 python:
   - '3.7'
+  - '3.8'
```

Comment:
Does this mean travis will run everything two times, once with 3.7 and once with 3.8?

Reply:
Yes, this will run the same job for both python versions. See https://travis-ci.org/raiden-network/raiden-contracts/builds/650339732